enabler/test/de/schildbach/pte/live/OntarioProviderLiveTest.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2010-2015 the original author or authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package de.schildbach.pte.live;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Date;
+
+import org.junit.Test;
+
+import de.schildbach.pte.NetworkProvider.Accessibility;
+import de.schildbach.pte.NetworkProvider.WalkSpeed;
+import de.schildbach.pte.OntarioProvider;
+import de.schildbach.pte.dto.Location;
+import de.schildbach.pte.dto.LocationType;
+import de.schildbach.pte.dto.NearbyLocationsResult;
+import de.schildbach.pte.dto.Product;
+import de.schildbach.pte.dto.QueryDeparturesResult;
+import de.schildbach.pte.dto.QueryTripsResult;
+import de.schildbach.pte.dto.SuggestLocationsResult;
+
+public class OntarioProviderLiveTest extends AbstractNavitiaProviderLiveTest
+{
+	public OntarioProviderLiveTest()
+	{
+		super(new OntarioProvider(secretProperty("navitia.authorization")));
+	}
+
+	@Test
+	public void nearbyStations() throws Exception
+	{
+		final NearbyLocationsResult result = queryNearbyStations(new Location(LocationType.STATION, "NA190"));
+		print(result);
+	}
+
+	@Test
+	public void nearbyStationsByCoordinate() throws Exception
+	{
+		final NearbyLocationsResult result = queryNearbyStations(Location.coord(45416667, -75683333));
+		print(result);
+	}
+
+	@Test
+	public void queryDepartures() throws Exception
+	{
+		final QueryDeparturesResult result = queryDepartures("NA190", false);
```

Comment:
Is that a valid station ID? Judging from the results of nearby stations, station IDs seem all to be prefixed with `OAW:SP:`.